js/views/login.html

```diff
@@ -1,6 +1,6 @@
 <div id="dialogWrapper">
   <div style="display:none" id="modalVideo" title="Welcome to Parakh" 
-       data-bind="ojComponent:{component: 'ojDialog', initialVisibility: 'hide',rootAttributes: { style: 'width: 600px; height: 400px;'}}">
+       data-bind="ojComponent:{component: 'ojDialog', initialVisibility: 'hide'}">
      <div class="oj-dialog-body video-align-center">
        	<img src="images/2016-12-11.gif">
```

Comment:
I thought that was login module so I named login.html. 

Where all login related views and event are handled.